pruner/light/window.go

```diff
@@ -0,0 +1,9 @@
+package light
+
+import (
+	"time"
+
+	"github.com/celestiaorg/celestia-node/pruner"
+)
+
+const Window = pruner.AvailabilityWindow(time.Second * 86400 * 30)
```

Comment:
non-blocking, but it would be nice to add a comment about how much time in days

Reply:
will add in follow up pr